File: ejercicios hechos/Java/estructuras alternativas/act4.java
/*Programa: act4.java
 * 
 *Propósito: Crea un programa que pida al usuario dos números y muestre su división si el segundo no es cero, o un mensaje de aviso en caso contrario.
 * 
 *Autor: Jose Manuel Serrano Palomo.
 * 
 *Fecha: 09/11/2019
 *
 *Análisis:
 * Tras pedir dos números mostramos su división pero si el divisor es 0 da error
 *
 *Diseño:
 * 1. Leer n1 y n2
 * 2. Si divisor es igual a 0 Escribir mensaje de error
 * 3. Sino mostrar resultado
 * Variables n1(dividendo) y n2(divisor)
*/
 
public class plantilla {
    public static void main(String[] args) {
        System.out.println("Programa que divide dos números");
        System.out.println("-------------------------------");
        Scanner s = new Scanner(System.in);
        int n1;
        int n2;

        System.out.print("Dime el primer numero: ");
        n1 = s.nextInt();
        System.out.print("Dime el segundo número: ");
        n2 = s.nextInt();

        if (n2==0) {
        System.out.println("Error, no se puede dividir entre 0");
        } else {
        System.out.println("El resultado es "+n1/n2);
        }
    }
}

File: ejercicios terminados/Python/estructuras alternativas/act4.py
#Programa: act4.py
#Propósito: pedir 2 numeros a un usuario y mostrar su división si el segundo no es 0 o un mensaje en caso contrario
#Autor: Jose Manuel Serrano Palomo.
#Fecha: 17/10/2019
#
#Variables a usar:
#n1 y n2 numeros a dividir
# 
#Algoritmo:
# LEER n1 y n2
# dividir n1 entre n2
# si n2 no es igual a 0 mostrar resultado
# por el contrario mensaje de error

print("programa para dividir dos numeros")
print("----------------------------------\n")

# Leemos datos
n1 = int(input("Introduce el primer número: "))
n2 = int(input("Introduce el segundo número: "))

# Hacemos los cálculos y los mostramos

if n2 == 0:
    print("El número introducido en el divisor no puede ser 0")
else:
    print("El resultado es: ",n1/n2)
